ksqldb-engine/src/main/java/io/confluent/ksql/execution/json/PlanJsonMapper.java

```diff
@@ -44,7 +44,7 @@
       .enable(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES)
       .enable(DeserializationFeature.FAIL_ON_NULL_CREATOR_PROPERTIES)
       .enable(DeserializationFeature.FAIL_ON_INVALID_SUBTYPE)
-      .setSerializationInclusion(Include.NON_EMPTY);
+      .setSerializationInclusion(Include.NON_NULL);
```

Comment:
Need to change this to preserve `Optional.empty()` (other wise it will be converted to `null` what we don't want).